lib/use_cases/fetch_pull_requests.rb

```diff
@@ -5,21 +5,41 @@ def initialize(gateway: Gateways::PullRequest.new)
     end
 
     def execute
-      pull_requests = gateway.execute
-
-      pull_requests.map do |result|
+      pull_request_hash = gateway.execute.map do |result|
         {
           application_name: result.application_name,
           title: result.title,
           url: result.url,
-          open_since: result.open_since,
-          version: result.version
+          open_since: result.open_since
         }
       end
+
+      split_summarised_pull_requests(pull_request_hash)
     end
 
   private
 
+    SUMMARISED_PR_TITLE_MATCH = 'Bump (\S+) and (\S+)'.freeze
```

Comment:
Does this regex match:
``` Bump Adrian and Ryan and Dan ```

What format gets used when 3 gems are bumped?

Reply:
`Bump Adrian, Ryan and Dan` is the format used - which this won't match. We'll update it to be more generic